docs/building-a-form/available-widgets.md

```diff
@@ -0,0 +1,112 @@
+# Available widgets
+
+These widgets are included in the us-forms-system by default. Set these widgets in a config file while building your form.
+
+Some widgets are associated with particular schema types or formats. There are additional widgets available, but the us-forms-system uses definitions they're included in, rather than just the field.
+
+Widgets from the [react-jsonschema-form library](./about-the-us-forms-system-library#understanding-react-jsonschema-form-rjsf) include string mappings. Widgets created specifically for US Forms System do not have mappings, and therefore must be specified by passing the component for the widget directly to the config. To include such widgets, import the widget at the top of the file:
```

Comment:
@dmethvin-gov - Here's where I pared down information quite a bit, since it had gotten more specific. Let me know what you think.

Reply:
Do we want to include information from my comment above about the purpose of widgets, which is just to figure out which type of HTML form element to render?

Reply:
Definitely. Will add. 👍

Reply:
@annekainicUSDS: The short description from the README is also included at the top of this page, so it's now there. Is that ok? Is there anything I should modify in this particular paragraph?